src/main.ts

```diff
@@ -80,6 +81,8 @@ export default class CopilotPlugin extends Plugin {
 
     this.registerView(CHAT_VIEWTYPE, (leaf: WorkspaceLeaf) => new CopilotView(leaf, this));
 
+    this.initActiveLeafChangeHandler();
```

Comment:
It registers an event listener for the "active-leaf-change" event but does not remove it during the plugin's onunload lifecycle. This can lead to memory leaks or multiple event handlers being registered if the plugin is reloaded or reinitialized multiple times.